packages/backend/src/ee/authentication/middlewares.ts

```diff
@@ -5,17 +5,17 @@ import {
     getErrorMessage,
     OrganizationMemberRole,
     ScimError,
-    ServiceAccountScope,
     type MemberAbility,
 } from '@lightdash/common';
+import {
+    ScimScope,
+    ServiceAccountScope,
+} from '@lightdash/common/src/ee/serviceAccounts/scopes';
 import { RequestHandler } from 'express';
 import { ServiceAccountService } from '../services/ServiceAccountService/ServiceAccountService';
 
 const getRoleForScopes = (scopes: ServiceAccountScope[]) => {
```

Comment:
why does this mapping live here? I'd expect it closer to the CASL files

Reply:
I don't think there's a reason. Can definitely move it.